docs/pages/roadmap.js

```diff
@@ -0,0 +1,676 @@
+    'link': '',
+  },
+  {
+    'task': 'Form autofix',
+    'deadline': 'Quarter 1',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description':
+      'Develop a lint rule to automatically convert <form>​ instances to our upcoming <Form>​ component',
+    'link': '',
+  },
+  {
+    'task': 'Box duplicate props autofix',
+    'deadline': 'Quarter 1',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description': 'Develop a lint rule to automatically combine props on Box where possible.',
+    'link': '',
+  },
+  {
+    'task': 'Generalized prop naming autofix',
+    'deadline': '',
+    'status': 'ok',
+    'platform': 'Web',
+    'description':
+      'Develop a lint rule to automatically rename props of Gestalt component instances.',
+    'link': '',
+  },
+  {
+    'task': 'Generalized prop value change autofix',
+    'deadline': '',
+    'status': 'ok',
+    'platform': 'Web',
+    'description':
+      'Develop a lint rule to automatically change prop values of Gestalt component instances.',
+    'link': '',
+  },
+  {
+    'task': 'Testing helpers library',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description':
+      'Develop infrastructure to support Gestalt component integration testing in Cypress.',
+    'link': '',
+  },
+  {
+    'task': 'Visual diff testing',
+    'deadline': '',
+    'status': 'inProgress',
+    'platform': 'Web',
+    'description':
+      'Ship visual diff tests for all Gestalt components to guard against UI regressions.',
+    'link': '',
+  },
+];
+
+const inProgress = tasks.filter((x) => x.status === 'inProgress');
+
+const future = tasks.filter((x) => x.status === 'unstarted');
+
+const complete = tasks.filter((x) => x.status === 'ok');
+
+const LockIcon = function LockIcon({ size }: {| size: 12 | 14 |}) {
+  return (
+    <Tooltip text="Access is restricted to Pinterest employees" accessibilityLabel="" inline>
+      <TapArea rounding="circle" fullWidth={false}>
+        <Icon
+          accessibilityLabel="Access is restricted to Pinterest employees"
+          icon="lock"
+          size={size}
+        />{' '}
+      </TapArea>
+    </Tooltip>
+  );
+};
+
+function Task({
+  text,
+  description,
+  delivery,
+  platform,
+}: {|
+  text: string,
+  description: string,
+  delivery: string,
+  platform: string,
+|}) {
+  const title = platform ? `${text} (${platform})` : text;
+
+  const desc = (
+    <Box marginTop={-6}>
+      <Flex direction="column" gap={2}>
+        <Text>{description}</Text>
+        {delivery ? <Text size="100">Scheduled for {delivery}, 2022</Text> : null}
+      </Flex>
+    </Box>
+  );
+
+  return <MainSection.Subsection title={title}>{desc}</MainSection.Subsection>;
+}
+
+export default function RoadmapPage(): Node {
+  const theme1Items = inProgress.map((obj) => (
+    <Task
+      key={obj.task}
+      text={obj.task}
+      description={obj.description}
+      delivery={obj.deadline}
```

Comment:
nit: would be slightly easier to follow if delivery were also called deadline, and text were called task, just so they match up 1 to 1

Reply:
Fixed!